metis/bin/metis_client

```diff
@@ -1194,7 +1203,11 @@ EOT
       unless files.empty?
         # compute md5sums
         file_md5s = files.map.with_index do |file,i|
-          print "Computing md5s - #{progress_bar(i.to_f / files.length)} - #{i} of #{files.length}\r"
+          if STDOUT.isatty
+            print "Computing md5s - #{progress_bar(i.to_f / files.length)} - #{i} of #{files.length}\r"
+          else
+            puts "Computing md5s - #{progress_bar(i.to_f / files.length)} - #{i} of #{files.length}"
```

Comment:
Won't this be spammy?

Reply:
Absolutely, but like I said above, Nuke is a destructive operation so verbosity is good, no? This is just an mdt check though so it may be OK to only print completion